auth.go

```diff
@@ -24,6 +24,8 @@ const (
 	sessionTokenLength = 30
 
 	defaultSessionExpirySeconds = 30 * 24 * 3600
+
+	tickerTime = 5
```

Comment:
Time constants must either have a unit in their name, such as "tickerTimeSeconds", or (preferably) be strongly typed, such as 
```go
const tickerTime = 5 * time.Second
```
Also, this variable needs a better name. Nothing tells us that it is the ticker interval for the LDAP identities merge.